ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -1112,6 +1365,68 @@ nms (GArray * results, gfloat threshold)
   } \
   break
 
+
+/**
+ * @brief C++-Template-like box location calculation for Tensorflow model
+ * @param[in] bb The configuration, "bounding_boxes"
+ * @param[in] data palm detection configuration, "properties_MP_PALM_DETECTION"
+ * @param[in] type The tensor type of inputptr
+ * @param[in] typename nnstreamer enum corresponding to the type
+ * @param[in] scoreinput Input Tensor Data (Detection scores)
+ * @param[in] boxesinput Input Tensor Data (Boxes)
+ * @param[in] config Tensor configs of the input tensors
+ * @param[out] results The object returned. (GArray with detectedObject)
+ */
+#define _get_objects_mp_palm_detection(bb, data, _type, typename, scoreinput, boxesinput, config, results) \
+  case typename: \
+  { \
+    int d_; \
+    _type * scores_ = (_type *) scoreinput; \
+    _type * boxes_ = (_type *) boxesinput; \
+    guint i_width_ = bb->i_width; \
+    guint i_height_ = bb->i_height; \
+    int num_ = bb->max_detection; \
+    size_t boxbpi_ = config->info.info[0].dimension[0]; \
+    results = g_array_sized_new (FALSE, TRUE, sizeof (detectedObject), num_); \
+    for (d_ = 0; d_ < num_; d_++) { \
+      _type y_center, x_center, h, w; \
+      _type ymin, xmin; \
+      int y, x, width, height; \
+      detectedObject object; \
+      _type score = scores_[d_]; \
+      _type * box = boxes_ + boxbpi_ * d_; \
+      anchor *a = &g_array_index (data->anchors, anchor, d_); \
+      score = MAX(score, -100.0f); \
```

Comment:
CI complained this for comparing float with other types (https://github.com/nnstreamer/nnstreamer/runs/6104768829?check_suite_focus=true#step:6:290).
May the `score` be float type could be better?

Reply:
thanks.
It changed to use `gfloat` type.

Reply:
thanks.
It changed to use gfloat type.